src/Constants.php

```diff
@@ -379,4 +379,30 @@ class Constants
                 ],
             ],
     ];
+
+    // Dashboard will be array of ads to show more ads if needed in future
+    const ADS_CONTENT = [
+        "dashboard" => [
+            [
+                "heading" => "AI/Machine Translation",
+                "content" => "Translate more content faster with AI.",
+                "linkText" => "Try it now",
+                "link" => self::CTA_AI_ML_TRANSLATION,
+            ]
```

Comment:
We should not display an ad for AI/MT yet.

Reply:
Done